tests/test_quoting.py

```diff
@@ -455,3 +459,130 @@ def test_quoter_path_with_plus(quoter):
 def test_unquoter_path_with_plus(unquoter):
     s = "/test/x+y%2Bz/:+%2B/"
     assert "/test/x+y+z/:++/" == unquoter(unsafe="+")(s)
+
+
+@given(safe=st.text(), protected=st.text(), qs=st.booleans(), requote=st.booleans())
+def test_fuzz__PyQuoter(safe, protected, qs, requote):
+    _PyQuoter(safe=safe, protected=protected, qs=qs, requote=requote)
+
+
+@given(ignore=st.text(), unsafe=st.text(), qs=st.booleans())
+def test_fuzz__PyUnquoter(ignore, unsafe, qs):
+    _PyUnquoter(ignore=ignore, unsafe=unsafe, qs=qs)
```

Comment:
Are these just for testing that no exceptions happen? Perhaps stick `assert` in front of the invocations so that some linters wouldn't be surprised? Also, could you add docstrings showing what these tests check?

Reply:
Yes, thats exactly whats going on